<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>restaurant_auth</class>
 <widget class="QWidget" name="restaurant_auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restaurant Authentication</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 0, 0);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
   <rect>
    <x>50</x>
    <y>50</y>
    <width>800</width>
    <height>600</height>
   </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,20);
border-radius: 10px;</string>
   </property>
   
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Restaurant Dashboard</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 24pt &quot;Segoe UI&quot;;
text-align: center;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
     <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to your restaurant management portal!</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;
text-align: center;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   
   <!-- Restaurant Information Section -->
   <widget class="QGroupBox" name="restaurantInfoGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>700</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Restaurant Information</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    
    <widget class="QLabel" name="restaurantNameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Restaurant Name:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QLineEdit" name="restaurantNameEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid rgb(200, 200, 200);
border-radius: 3px;
color: rgb(0, 0, 0);
font: 9pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QLabel" name="restaurantTypeLabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Restaurant Type:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QComboBox" name="restaurantTypeCombo">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid rgb(200, 200, 200);
border-radius: 3px;
color: rgb(0, 0, 0);
font: 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Fast Food</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Iranian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>International</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cafe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pizza</string>
      </property>
     </item>
    </widget>
    
    <widget class="QLabel" name="restaurantAddressLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QLineEdit" name="restaurantAddressEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>480</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid rgb(200, 200, 200);
border-radius: 3px;
color: rgb(0, 0, 0);
font: 9pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QLabel" name="restaurantDescLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QTextEdit" name="restaurantDescEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>480</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid rgb(200, 200, 200);
border-radius: 3px;
color: rgb(0, 0, 0);
font: 9pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="save_info_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
color: rgb(255, 255, 255);
font: 700 12pt 'Segoe UI';
border-bottom: 1px solid rgba(255,0,0,200);
border-top: 1px solid rgba(255,0,0,200);
border-left: 1px solid rgba(255,0,0,200);
border-right: 1px solid rgba(255,0,0,200);</string>
     </property>
     <property name="text">
      <string>Save Restaurant Info</string>
     </property>
    </widget>
   </widget>
   
   <!-- Status Section -->
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>700</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Checking restaurant status...</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
text-align: center;
background-color: rgba(0,0,0,50);
border-radius: 5px;
padding: 10px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   
   <!-- Menu Management Section -->
   <widget class="QGroupBox" name="menuManagementGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>700</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu Management</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    
    <widget class="QTextEdit" name="infoTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,90);
border: 1px solid rgb(200, 200, 200);
border-radius: 5px;
color: rgb(0, 0, 0);
font: 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Restaurant Management Features:

• Add, edit, and delete menu items
• Set food categories and prices
• Manage restaurant information
• View customer orders
• Track restaurant performance

Please ensure your restaurant information and menu are properly set up.</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="continue_button">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>120</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
color: rgb(255, 255, 255);
font: 700 12pt 'Segoe UI';
border-bottom: 1px solid rgba(255,0,0,200);
border-top: 1px solid rgba(255,0,0,200);
border-left: 1px solid rgba(255,0,0,200);
border-right: 1px solid rgba(255,0,0,200);</string>
     </property>
     <property name="text">
      <string>Continue to Menu</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="setup_menu_button">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>75</y>
       <width>120</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
color: rgb(255, 255, 255);
font: 700 12pt 'Segoe UI';
border-bottom: 1px solid rgba(255,0,0,200);
border-top: 1px solid rgba(255,0,0,200);
border-left: 1px solid rgba(255,0,0,200);
border-right: 1px solid rgba(255,0,0,200);</string>
     </property>
     <property name="text">
      <string>Setup Menu Now</string>
     </property>
    </widget>
   </widget>
   
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: none;
color: rgb(255, 255, 255);
font: 700 12pt 'Segoe UI';
border-bottom: 1px solid rgba(255,0,0,200);
border-top: 1px solid rgba(255,0,0,200);
border-left: 1px solid rgba(255,0,0,200);
border-right: 1px solid rgba(255,0,0,200);</string>
    </property>
    <property name="text">
     <string>Back to Dashboard</string>
    </property>
   </widget>
   
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 